arches/app/search/search.py

```diff
@@ -21,8 +21,9 @@
 import urllib.error
 import uuid
 import logging
+import warnings
 from datetime import datetime
-from elasticsearch import Elasticsearch, helpers
+from elasticsearch import Elasticsearch, helpers, ElasticsearchWarning
```

Comment:
`ElasticsearchWarning` was added in `elasticsearch` [7.11.0](https://newreleases.io/project/pypi/elasticsearch/release/7.11.0), but our `requirements.txt` current specifies `elasticsearch>=7.11.0,<8.0.0`, so we should update this accordingly.

Reply:
I've updated requirements.txt to `elasticsearch>=7.11.0,<8.0.0` as per your request.